tests/test_linters.py

```diff
@@ -341,15 +342,23 @@ def test_framework_operator_no_ops_imported(tmp_path, monkeypatch, import_line):
 @pytest.mark.parametrize(
     "charm_name, charm_module", [("foobar", "foobar.py"), ("foo-bar", "foo_bar.py")]
 )
-def test_framework_reactive_used_ok(tmp_path, import_line, charm_name, charm_module):
+def test_framework_reactive_used_ok(create_config, import_line, charm_name, charm_module):
     """The reactive framework was used.
 
     Parametrized args:
     - import_line: different ways to express the import
     """
-    # metadata file with needed name field
-    metadata_file = tmp_path / "metadata.yaml"
-    metadata_file.write_text(f"name: {charm_name}")
+    tmp_path = create_config(
+        {
+            "metadata.yaml": dedent(
+                f"""
+                name: {charm_name}
```

Comment:
Please ensure you're keeping the same charm as before. If you want a version that contains these extra fields, add a parametrize fixture to provide both.

Reply:
That's not possible. The old test case used an invalid `metadata.yaml` by specs. It's just because old charmcraft doesn't have these validations cause it can pass a single line of `name: {charm_name}`

Reply:
Ahh, I see — so if you put that charm into the old charmcraft it would still (correctly) error out? In that case I'm fine with this change :smile: